src/Monolog/Formatter/DatadogFormatter.php

```diff
@@ -3,28 +3,16 @@
 namespace sgoettsch\MonologDatadog\Formatter;
 
 use Monolog\Formatter\JsonFormatter;
+use Monolog\Level;
 use Monolog\Logger;
+use Monolog\LogRecord;
 use stdClass;
 
 class DatadogFormatter extends JsonFormatter
 {
-    protected $includeStacktraces = true;
-
-    /**
-     * Map Monolog\Logger levels to Datadog's default status type
-     */
-    private const DATADOG_LEVEL_MAP = [
-        Logger::DEBUG => 'info',
```

Comment:
`Logger::DEBUG` and the other constants are deprecated. Monolog now uses the `Level` enum. For PHP 8.0 compatibility, we are unable to use enum cases as array keys. This is only supported with PHP 8.2.